experiment/build/generate_cloudbuild.py

```diff
@@ -98,10 +150,11 @@ def create_cloudbuild_spec(image_templates,
         }
         step['args'] = [
             'build', '--tag',
-            posixpath.join(docker_registry, image_specs['tag']), '--tag',
-            get_experiment_tag_for_image(image_specs), '--cache-from',
-            get_experiment_tag_for_image(image_specs, tag_by_experiment=False),
-            '--build-arg', 'BUILDKIT_INLINE_CACHE=1'
+            _get_experiment_image_tag(image_specs), '--tag',
+            _get_gcb_image_tag(image_specs), '--tag',
```

Comment:
Should gcb break if you provide the same tag twice ?

Like _get_experiment_image_tag(image_specs) can be == _get_gcb_image_tag(image_specs) (can be seen in some tests). it probably makes sense to check if and not add a duplicate if it is the same tag.

Reply:
I'm pretty sure "it" (it would probably be docker that breaks) does not break.

Reply:
fine either way, nice to fix with a 2 line fix.

Reply:
I tested that it does not break. I don't see why we should add this logic. It just complicates things without providing any benefits.